cmd/manager/main.go

```diff
@@ -207,6 +207,8 @@ func init() {
 		"Select a namespace for the controller loop. If not set, uses ${POD_NAMESPACE} environment variable")
 	rootCmd.PersistentFlags().DurationVar(&flags.controllerOptions.ResyncPeriod, "resync", time.Second,
 		"Controller resync interval")
+	rootCmd.PersistentFlags().StringVar(&flags.controllerOptions.DomainSuffix, "domainSuffix", "cluster.local",
```

Comment:
The flag used by kubeadm to specify this is called `--service-dns-domain`. Shall we just use the same thing?